main.js

```diff
@@ -0,0 +1,67 @@
+let apis;
+let savedUsers = [];
+
+//load data from apis
+$('#data-btn').on('click', function () {
+  apis = new APIManager();
```

Comment:
you should make an instance of the APIManager only one time in this program, outside of the event listener